apps/web_delivery/test/controllers/status_controller_test.exs

```diff
@@ -3,6 +3,6 @@ defmodule WebDelivery.StatusControllerTest do
 
   test "GET /api", %{conn: conn} do
     conn = get conn, "/api"
-    assert json_response(conn, 200) == %{"status" => "OK"}
+    assert %{"status" => "OK"} === json_response(conn, 200)
```

Comment:
There are actually two changes there: (1.) from equality to match equality and (2.) yoda-style assertion. Yoda-style assertions are commonplace in Elixir because they offer the ability to easily switch between comparison and pattern-matching. This comes in _particularly_ handy for maps. For example if I have this assertion:
```elixir
assert %{foo: 1, bar: 2, baz: 3} === some_fun(times)
```

...and I decide to add some metadata to it, say by turning it into a struct or backing it with an Ecto schema, and that metadata is unrelated to my assertion, all I have to do is change match equality to match, like so:
```elixir
assert %{foo: 1, bar: 2, baz: 3} = some_fun(times)
```

In order to make tests read consistently I recommend _always_ using yoda-style assertions in tests.

***

P.S. The style convention for multiline structures still applies with yoda-style assertions. This is pulled from the Guardian code:
```elixir
assert %{
  deposit_type: "wire",
  check_id: nil,
  check_number: nil,
} = changeset.changes
```